webview/src/plots/components/ribbon/Errors.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styles from './styles.module.scss'
+import { PlotsState } from '../../store'
+import { Error } from '../../../shared/components/icons'
+import { setShowErrorsModal } from '../webviewSlice'
+
+export const Errors: React.FC = () => {
+  const dispatch = useDispatch()
+  const [errorsTotal, setErrorsTotal] = useState(0)
+  const errors = useSelector((state: PlotsState) => state.webview.plotErrors)
+
+  useEffect(() => {
+    let total = 0
+    for (const { revs } of errors) {
+      total += revs.length
+    }
+    setErrorsTotal(total)
+  }, [errors])
+
+  if (errors.length === 0) {
+    return
+  }
+
+  return (
+    <li className={styles.errors}>
```

Comment:
It does not make much sense for this to be an `<li/>` element. I know that it is inside of the ribbon, but it does not fit with the rest of the list. The ribbon should be a wrapper that wraps the list and the errors.

Reply:
Makes sense! I'll take care of this in a followup since the CSS will need to be adjusted and I don't want to accidentally break the ribbon.